content/en/blog/_posts/2023-04-03-improving-support-go-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Improving Support: Go Updates"
+date: 2023-04-03
+slug: improving-support-go-updates
```

Comment:
suggestions/corrections/improvements for the title/date/slug welcome... I seemed to have to backdate the date to get it to show up in the preview